src/main/java/frc/team449/_2022robot/cargo/spitterClass.java

```diff
@@ -0,0 +1,11 @@
+package frc.team449._2022robot.cargo;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.team449.generalInterfaces.SmartMotor;
+import org.jetbrains.annotations.NotNull;
+
+public class SpitterClass extends SubsystemBase {
```

Comment:
Just `Spitter` will do, no need for the `Class` suffix
```suggestion
public class Spitter extends SubsystemBase {
```